walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,48 @@
 
 package com.google.sps;
 
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.Comparator;
 import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    ArrayList<TimeRange> availableTimes = new ArrayList<>();
+    ArrayList<TimeRange> unavailableTimes = new ArrayList<>();
+    int meetingStartTime = TimeRange.START_OF_DAY;
```

Comment:
nit: move the declaration down to where it's being used. it makes code more readable when the variables you use are declared where you use them